test/entrypoint/srv_select/storage_init.lua

```diff
@@ -227,4 +228,100 @@ return function()
         unique = false,
         if_not_exists = true,
     })
+
+    if datetime_supported then
+        local datetime_format = {
+            {name = 'id', type = 'unsigned'},
+            {name = 'bucket_id', type = 'unsigned'},
+            {name = 'datetime_field', type = 'datetime'},
+        }
+
+
+        local datetime_nonindexed_space = box.schema.space.create('datetime_nonindexed', {
+            if_not_exists = true,
+            engine = engine,
+        })
+
+        datetime_nonindexed_space:format(datetime_format)
+
+        datetime_nonindexed_space:create_index('id_index', {
+            parts = { 'id' },
+            if_not_exists = true,
+        })
+
+        datetime_nonindexed_space:create_index('bucket_id', {
+            parts = { 'bucket_id' },
+            unique = false,
+            if_not_exists = true,
+        })
+
+
+        local datetime_indexed_space = box.schema.space.create('datetime_indexed', {
+            if_not_exists = true,
+            engine = engine,
+        })
+
+        datetime_indexed_space:format(datetime_format)
+
+        datetime_indexed_space:create_index('id_index', {
+            parts = { 'id' },
+            if_not_exists = true,
+        })
+
+        datetime_indexed_space:create_index('bucket_id', {
+            parts = { 'bucket_id' },
+            unique = false,
+            if_not_exists = true,
+        })
+
+        datetime_indexed_space:create_index('datetime_index', {
+            parts = { 'datetime_field' },
+            unique = false,
+            if_not_exists = true,
+        })
+
+
+        local datetime_pk_space = box.schema.space.create('datetime_pk', {
+            if_not_exists = true,
+            engine = engine,
+        })
+
+        datetime_pk_space:format(datetime_format)
+
+        datetime_pk_space:create_index('datetime_index', {
+            parts = { 'datetime_field' },
+            if_not_exists = true,
+        })
+
+        datetime_pk_space:create_index('bucket_id', {
+            parts = { 'bucket_id' },
+            unique = false,
+            if_not_exists = true,
+        })
+
+
```

Comment:
It was intentional since there is a hundred lines of code and it would be nice to visually separate one space setup from the other. But I can remove them, if you insist.